packages/app/api/listing/numbers.ts

```diff
@@ -0,0 +1,41 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { select } from "../../lib/db";
+
+export interface NumbersProps {
+  matched_beds: string;
+  available_beds: string;
+  requested_beds: string;
+}
+
+export type GetNumberList = {
+  ok: "ok";
+  numbers: NumbersProps;
+};
+
+async function getNumbers(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
**[pages/api](https://nextjs.org/docs/api-routes/introduction)** is a feature of next.js - it is not available at the expo, if you only need a type, then make a file for the type or copy it to the place where you need it